dmsghttp/util.go

```diff
@@ -0,0 +1,57 @@
+package dmsghttp
+
+import (
+	"context"
+	"net/http"
+	"time"
+
+	"github.com/skycoin/dmsg/direct"
+	"github.com/skycoin/dmsg/disc"
+
+	"github.com/skycoin/skycoin/src/util/logging"
+)
+
+// GetServers test
+func GetServers(ctx context.Context, dmsgDisc string, log *logging.Logger) (entries []*disc.Entry) {
+	dmsgclient := disc.NewHTTP(dmsgDisc, http.Client{})
+	ticker := time.NewTicker(time.Second * 10)
+	defer ticker.Stop()
+	for {
+		servers, err := dmsgclient.AvailableServers(ctx)
+		if err != nil {
+			log.WithError(err).Fatal("Error getting dmsg-servers.")
+		}
+		if len(servers) > 0 {
+			return servers
+		}
+		log.Warn("No dmsg-servers found, trying again in 1 minute.")
```

Comment:
Why are we replicating the code we have in dmsg discovery root command?

Reply:
In the root command of `dmsg-discovery` we get the `dmsg-servers` from the store directly. In util we get it from the URL. It is for the other services.